test/unit/account_test.rb

```diff
@@ -21,6 +21,15 @@ class AccountTest < ActiveSupport::TestCase
     user.account.destroy
   end
 
+  test "create a new account" do
```

Comment:
I've added another test to make sure account creation works when the provider decides not to use invite code. The test is copy&pasted from the original one.